device-registry/src/main/scala/org/genivi/sota/device_registry/GroupsResource.scala

```diff
@@ -72,8 +74,8 @@ class GroupsResource(namespaceExtractor: Directive1[Namespace], deviceNamespaceA
 
   def createGroupFromDevices(request: CreateGroupRequest, namespace: Namespace): Route = {
     val commonJsonDbIo = for {
-      info1 <- findByUuid(request.device1)
-      info2 <- findByUuid(request.device2)
+      info1 <- findByUuid(request.device1).map(removeIdNrs)
+      info2 <- findByUuid(request.device2).map(removeIdNrs)
```

Comment:
This method should also not be on the controller, should be somewhere else that knows about `removeIdNrs` and databases